src/Experiment/tests/BlankAssemblyTest.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit.Experimental;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using Xunit;
+
+namespace Experiment.Tests
+{
+    public class BlankAssemblyTest
+    {
+        [Fact]
+        public void GenerateBlankAssemblyReadToDisk()
+        {
+            AssemblyName _name = new AssemblyName("MyBlankAssembly");
+            _name.Version = new Version("4.3");
+            const String _fileName = "BlankAssembly.dll";
+            AssemblyBuilder assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly(_name, System.Reflection.Emit.AssemblyBuilderAccess.Run);
+            //We need to create module because even a blank assembly has one module.
+            assemblyBuilder.DefineDynamicModule(_name.Name);
+            assemblyBuilder.Save(_fileName);
+            TryLoadAssembly(_fileName,_name);//It seems that sometimes ILSpy will point out a DLL is malformed but MetadataReader doesn't throw an error (MetadataLoadContext rather?) .
```

Comment:
It would be good if TryLoadAssembly returns back the `Assembly` object, so that you can then assert things in the test after loading it. For example: it would be good in this test that we validate that the assembly name and version are what we expect